scripts/updatebreaches.js

```diff
@@ -35,6 +35,7 @@ if (
       BreachDate: breach.BreachDate,
       PwnCount: breach.PwnCount,
       AddedDate: breach.AddedDate,
+      BreachDataClasses: breach.DataClasses,
```

Comment:
Maybe consider calling this simply "DataClasses"? That would keep the parity since everything else follows the HIBP names.

Reply:
Dang, you're right ... but I already updated the Remote Settings schema with this new field name. I imagine it's harder to update a field than to add a new one? I'll check it out tomorrow though.

Reply:
Updated name to just `DataClasses` and filed https://github.com/mozilla-services/remote-settings-permissions/pull/68 for the corresponding change to the RS schema.